connectors/sources/google_cloud_storage.py

```diff
@@ -296,14 +276,19 @@ async def fetch_blobs(self, buckets):
             Dictionary: Contains the list of fetched blobs from Google Cloud Storage.
         """
         for bucket in buckets.get("items", []):
-            async for blob in self._google_storage_client.api_call(
-                resource="objects",
-                method="list",
-                full_response=True,
-                bucket=bucket["id"],
-                userProject=self._google_storage_client.user_project_id,
-            ):
-                yield blob
+            try:
+                async for blob in self._google_storage_client.api_call(
+                    resource="objects",
+                    method="list",
+                    full_response=True,
+                    bucket=bucket["id"],
+                    userProject=self._google_storage_client.user_project_id,
+                ):
+                    yield blob
+            except Exception as exception:
```

Comment:
Let's handle the access error separately. It will be much easier to find it in the logs and it will look much better in the UI.